js/index.js

```diff
@@ -40,3 +40,79 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+let navBar = document.querySelectorAll('a');
+let navB = Array.from(navBar);
+navB[0].textContent = 'Services'
+navB[1].textContent = 'Product'
+navB[2].textContent = 'Vision'
+navB[3].textContent = 'features'
+navB[4].textContent = 'About'
+navB[5].textContent = 'Contact'
+
+
+let naviGator = document.querySelector('nav');
+let add1 = document.createElement('a');
```

Comment:
Remember to use `const` rather then `let` unless there is a reason to be updating the value